.github/workflows/update-changelog.yml

```diff
@@ -34,5 +34,5 @@ jobs:
       - name: Push changes
         run: |
           git add CHANGELOG.md
-          git commit -m "Update CHANGELOG.md [skip ci]"
+          git commit -m "post-release changes [auto]"
```

Comment:
Capitalize p: `Post-release changes`. Should it be `Post-release updates`?

Reply:
Done.